### PR TITLE
Renamed mr_proc mentions

### PR DESCRIPTION
**Pull request by nikhil153**
Updated following comment mentions / variable names 

1. `mr_proc` --> `nipoppy` 
2. `mr_proc_manifest` --> manifest 
3. `mr_proc_root` --> `dataset_root`

### CODE REVIEW

**Review by michellewang — CHANGES_REQUESTED** [review]
I did a quick search for "mr_proc" in your branch (https://github.com/search?q=repo%3Anikhil153%2Fnipoppy+mr_proc&type=code) and found some code/text that wasn't renamed:
- 4 in [`trackers/run_tracker.py`](https://github.com/nikhil153/nipoppy/blob/6980633f168e11a0476dccb9bb35d5156503ba6e/trackers/run_tracker.py#L35)
- 1 in [`docs/README.md`](https://github.com/nikhil153/nipoppy/blob/6980633f168e11a0476dccb9bb35d5156503ba6e/docs/README.md?plain=1#L27)
- 1 in [`workflow/dicom_org/shell_utils/download_from_loni.sh`](https://github.com/nikhil153/nipoppy/blob/6980633f168e11a0476dccb9bb35d5156503ba6e/workflow/dicom_org/shell_utils/download_from_loni.sh#L12)

The rest looks good!

File: trackers/run_tracker.py [hunk 1]
```diff
@@ -28,11 +28,11 @@ def run(global_config_file, dash_schema_file, pipelines, run_id=1):
     for pipeline in pipelines:
         pipe_tracker = tracker(global_config_file, dash_schema_file, pipeline) 
             
-        mr_proc_root_dir, session_ids, version = pipe_tracker.get_global_configs()
+        dataset_root, session_ids, version = pipe_tracker.get_global_configs()
         schema = pipe_tracker.get_dash_schema()
         tracker_configs = pipeline_tracker_config_dict[pipeline]
 
-        mr_proc_manifest = f"{mr_proc_root_dir}/tabular/mr_proc_manifest.csv"
+        mr_proc_manifest = f"{dataset_root}/tabular/mr_proc_manifest.csv"
```
Comment (michellewang) [inline trackers/run_tracker.py:35]:
```suggestion
        manifest = f"{dataset_root}/tabular/manifest.csv"
```

File: trackers/run_tracker.py [hunk 2]
```diff
@@ -28,11 +28,11 @@ def run(global_config_file, dash_schema_file, pipelines, run_id=1):
...
         manifest_df = pd.read_csv(mr_proc_manifest)
```
Comment (michellewang) [inline trackers/run_tracker.py:36]:
```suggestion
        manifest_df = pd.read_csv(manifest)
```

**Review by michellewang — COMMENTED** [review]
I added suggestions for the ones that were missed, but there are some that I can't change (too far from code you changed):
- L103 in `trackers/run_tracker.py`
- L27 in `docs/README.md`
- L12 `workflow/dicom_org/shell_utils/download_from_loni.sh`

**Comment by nikhil153** [comment]
@michellewang - Fixed!

**Review by michellewang — APPROVED** [review]
👍

**Pull request merged**